src/ert/storage/local_storage.py

```diff
@@ -453,12 +458,17 @@ def _migrate(self) -> None:
                 self._acquire_lock()
                 self._add_migration_information(0, _LOCAL_STORAGE_VERSION, "block_fs")
             elif version < _LOCAL_STORAGE_VERSION:
-                migrations = list(enumerate([to2, to3, to4, to5], start=1))
+                migrations = list(enumerate([to2, to3, to4, to5, to6], start=1))
                 for from_version, migration in migrations[version - 1 :]:
                     migration.migrate(self.path)
                     self._add_migration_information(
                         from_version, from_version + 1, migration.info
                     )
+
+                for exp in self._experiments.values():
```

Comment:
Why are these here?

Reply:
To clear the cache from `.observations` and `.observation_keys`, as it is accessed before migrations are done, leaving it in an "outdated" state due to the @property caching. It is documented from python 3.9 and onwards.

 ref:
https://stackoverflow.com/questions/62662564/how-do-i-clear-the-cache-from-cached-property-decorator

Reply:
Ahh, that was not obvious to me, perhaps add a comment?